File: src/main/java/io/github/gravetii/scene/start/StartControlsController.java
package io.github.gravetii.scene.start;

import io.github.gravetii.controller.FxController;
import io.github.gravetii.controller.MenuBarController;

import javafx.fxml.FXML;

public class StartControlsController implements FxController {
  private final MenuBarController ref;

  public StartControlsController(MenuBarController ref) {
    this.ref = ref;
  }

  @FXML
  private void onPlayBtnClick() throws Exception {
    this.ref.newGame();
  }

  @FXML
  private void onChangeThemeBtnClick() throws Exception {
    this.ref.editTheme();
  }

  @FXML
  private void onWhatIsItBtnClick() throws Exception {
    this.ref.whatIsIt();
  }
}

File: src/main/kotlin/io/github/gravetii/controller/StartControlsController.kt
package io.github.gravetii.controller

import javafx.fxml.FXML

class StartControlsController(private val ref: MenuBarController) : FxController {

    @FXML
    fun onPlayBtnClick() = ref.newGame()

    @FXML
    fun onChangeThemeBtnClick() = ref.editTheme()

    @FXML
    fun onWhatIsItBtnClick() = ref.whatIsIt()

}